### PR TITLE
Harness for Fedora 33

### PR DESCRIPTION
**Pull request by StykMartin**
You welcome

### CODE REVIEW

**Comment by StykMartin** [comment]
@danrodrig ^

**Comment by danrodrig** [comment]
@StykMartin thank you, sir

File: yum-repos.conf [hunk 1]
```diff
@@ -345,15 +345,30 @@ beaker-system-scan = beaker-system-scan beaker-system-scan-debuginfo beaker-syst
...
 name = harness
 testing-name = harness-testing
-distro = Fedorarawhide
+distro = Fedora33
 source = brew
 arches = x86_64 ppc64le s390x aarch64
 tag = eng-fedora-33
 testing-tag = eng-fedora-33-candidate
 
+[harness-f33.packages]
```
Comment (cbouchar) [inline yum-repos.conf:357]: Shouldn't there be changes for [client-frawhide] [client-frawhide] too?

Comment (mdujava) [reply yum-repos.conf:357]: it's on line 357

Comment (StykMartin) [reply yum-repos.conf:357]: We are speaking about harness. No

Comment (cbouchar) [reply yum-repos.conf:357]: Ah My bad!  What's above is for BEAKER client.

**Comment by StykMartin** [comment]
Make sure you have everything before you will merge this otherwise you will blow up magnificently

**Comment by cbouchar** [comment]
> Make sure you have everything before you will merge this otherwise you will blow up magnificently

1) What do you mean by "everything"?  FYI Daniel(on PTO) has requested f34 branch.

2) I'm still trying to come up to speed on how this file ties into the release/upgrade procedure.  Before this can be merged, I believe I first need to update the following files in restraint with eng-fedora-34 once f34 branch is created.
 rel-eng/releasers.conf
 ./.packit.yaml 
./.github/workflows/review-checks.yml
./.github/workflows/merge-checks.yml
Then I cut another release and we perform another upgrade.  yes?  Do we pop the earliest fedora-31 off the list as we push fedora-34 onto the list?

**Comment by StykMartin** [comment]
> > Make sure you have everything before you will merge this otherwise you will blow up magnificently
> 
>     1. What do you mean by "everything"?  FYI Daniel(on PTO) has requested f34 branch.
> 

All builds have to be done before merging this.

>     2. I'm still trying to come up to speed on how this file ties into the release/upgrade procedure.  Before this can be merged, I believe I first need to update the following files in restraint with eng-fedora-34 once f34 branch is created.
>        rel-eng/releasers.conf
>        ./.packit.yaml
>        ./.github/workflows/review-checks.yml
>        ./.github/workflows/merge-checks.yml
>        Then I cut another release and we perform another upgrade.  yes?  Do we pop the earliest fedora-31 off the list as we push fedora-34 onto the list?

No. This has nothing to do with rest.

**Comment by cbouchar** [comment]
> > > Make sure you have everything before you will merge this otherwise you will blow up magnificently
> > 
> > 
> > ```
> > 1. What do you mean by "everything"?  FYI Daniel(on PTO) has requested f34 branch.
> > ```
> 
> All builds have to be done before merging this.

Which build are you referring to as Renan mentioned something about Jenkins?  I think you're asking me to go through the release procedure again?  If that is correct, your changes are referring to eng-fedora-34.  Doesn't rel-eng/releasers.conf need to change since it does not reference eng-fedora-34?

> 
> > ```
> > 2. I'm still trying to come up to speed on how this file ties into the release/upgrade procedure.  Before this can be merged, I believe I first need to update the following files in restraint with eng-fedora-34 once f34 branch is created.
> >    rel-eng/releasers.conf
> >    ./.packit.yaml
> >    ./.github/workflows/review-checks.yml
> >    ./.github/workflows/merge-checks.yml
> >    Then I cut another release and we perform another upgrade.  yes?  Do we pop the earliest fedora-31 off the list as we push fedora-34 onto the list?
> > ```
> 
> No. This has nothing to do with rest.

**Review by cbouchar — COMMENTED** [review]
Since we are removing harness-f31 from other code, shouldn't it be removed here as well from lines 320 to 332?

**Comment by StykMartin** [comment]
> Since we are removing harness-f31 from other code, shouldn't it be removed here as well from lines 320 to 332?

Sure, I created this PR long time ago.
F31 was kinda fine back then :)

**Comment by cbouchar** [comment]
Since removal of f31 has not occurred, it will be done in a separate commit since it is not essential for this changeset.

**Review by cbouchar — APPROVED** [review]

**Pull request merged**